packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -1073,4 +1145,145 @@ The Authenticator object defines the following properties:
         }
     }
 }
-```
\ No newline at end of file
+```
+
+### Authenticator Method object
+
+#### Authenticator Method properties
+
+| Property | Type | Description  | Applies to Authenticator Method type |
+| -------- | ---- | ------------ | ------------------------------------ |
+| `status` | String (Enum) | The status of the authenticator method. Possible values: `ACTIVE` or `INACTIVE` | All Authenticator Methods |
+| `type` | String (Enum) | The type of authenticator method. Possible values: `cert`, `duo`, `email`, `idp`, `otp`, `password`, `push`, `security_question`, `signed_nonce`, `sms`, `totp`, `voice`, or `webauthn`  |  All Authenticator Methods |
+| `_links` | [JSON HAL](https://tools.ietf.org/html/draft-kelly-json-hal-06) | Link relations for this Authenticator Method object  |  All Authenticator Methods |
+| `settings` | [Authenticator Method Settings object](#authenticator-method-settings-object) | Specific settings for the authenticator method  | `otp`, `push`, `signed_nonce`, `totp`, `webauthn` |
+
+### Authenticator Method Settings object
+
+#### Authenticator Method Settings properties
+
+| Property | Type | Description  | Applies to Authenticator Method type |
+| -------- | ---- | ------------ | ------------------------------------ |
+| `algorithms` | String (Enum) | Algorithms supported. Possible values: `RS256`, `ES256`, `HMACSHA1`, `HMACSHA256`, or `HMACSHA512` | `otp`, `totp`, `push`, `signed_nonce` |
+| `attachment` | String (Enum) | Method attachment. Possible values: `ANY`, `BUILT_IN`, or `ROAMING`  | `webauthn` |
+| `keyProtection` | String (Enum) | Indicates whether you must use a hardware key store for `push` and `signed_nonce` methods or if any type of key store is allowed. Supported values: `ANY` or `HARDWARE`  | `push`, `signed_nonce`|
+| `protocol` | String (Enum) | The protocol used. Possible values: `SYMANTEC`,`TOTP`, or `YUBICO` | `otp` |
+| `timeIntervalInSeconds` | Integer | Time interval for TOTP in seconds | `totp` |
```

Comment:
Hm, OK so this is a bit weird -- apparently TOTP is deprecated per https://github.com/okta/okta-core/blob/209a962ebc728cfb93bbea903430045c26a102d8/components/core/factors/model/src/main/java/com/okta/services/factors/iface/types/AuthenticationMethodType.java#L48-L53.

For Custom OTP factor, the AuthenticationMethodType is OTP, for example (per https://github.com/okta/okta-core/blob/209a962ebc728cfb93bbea903430045c26a102d8/components/core/factors/model/src/main/java/com/okta/services/factors/iface/types/AuthenticationMethodType.java#L58 and https://github.com/okta/okta-core/blob/209a962ebc728cfb93bbea903430045c26a102d8/components/web/factors/api/src/main/java/com/okta/services/factors/iface/dto/AuthenticatorCatalog.java#L50.

However, its method settings are of type TotpMethodSettings, because it's indeed time-based. I'm not 100% sure what this doc is going for, but it could be the case that, if we're talking about AuthenticatorMethodType, that there shouldn't be TOTP here but only OTP.

Reply:
TOTP is the method for okta_verify

It was created before we made the more general OTP method

@mpeng-okta But there is a TotpMethodSettings.java used by TotpMethodConfig

Reply:
We synced on slack and I get it now, yup.

So it looks like both OTP method config's settings and TOTP method config's settings support:
- timeIntervalInSeconds
- encoding
- algorithm
- passCodeLength

And then OTP method config's own TotpMethodSettings supports:
- acceptableAdjacentIntervals
- protocol

Reply:
Updated otp and totp settings.